### DISCUSSION TITLE
Dock goes to the secondary monitor in dual monitors

### DISCUSSION THREAD

**Discussion by akmalkadi** [linuxdeepin/developer-center#3048] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi,
I am using Deepin 20.1 and I have two monitors (the main is HDMI and the second is DP). I noticed most of the time that the dock goes to the secondary instead of staying in the main screen. I have sometimes to play in the settings to move it back but it again moves back to the secondary screen. Please see the attached images.

![WhatsApp Image 2021-02-05 at 12 09 51 PM](https://user-images.githubusercontent.com/1953002/107067465-518b0300-67ad-11eb-9bf5-4791694fa5cc.jpeg)
![image](https://user-images.githubusercontent.com/1953002/107067509-5f408880-67ad-11eb-89bf-945c45bac95b.png)

**Comment by tabuz** [comment]
I suffer the same issue. Try to click bottom of main screen while clicking shift/crtl - dock will flicker between monitors.

**Comment by jonalxh** [comment]
Same issue here, the dock is changing between main and secondary screens along usage.

**Comment by ssk-wh** [comment]
Can you provide a gif or video of this question？thank you very much!

**Comment by jonalxh** [comment]
Hi. As you can see in the [following gif ](https://drive.google.com/file/d/1Iscs43N2V5DHLU0TrOSp_-p3Ud2f_7qm/view?usp=sharing)I have chosen my left screen, but dock is almost always is shown in the secondary.

**Comment by 0yanghongwei** [comment]
Hi. The dock can be moved in multi-screen mode. When the mouse stays at the bottom for a period of time, the dock will appear on the current screen. When switching between the main and the secondary screen, the position of the appear will not change.

**Comment by nunesvictor** [comment]
> Hi. The dock can be moved in multi-screen mode. When the mouse stays at the bottom for a period of time, the dock will appear on the current screen. When switching between the main and the secondary screen, the position of the appear will not change.

Huh, I guess this is a feature instead of an issue. Still annoying tho, is there any way to disable it?

**Comment by akmalkadi** [comment]
I don't know if this issue was fixed or still. Anyway, I am not interested in Deepin anymore. So, please feel free to close this issue whenever it gets solved.

**Reply by BLumia** [reply]
I'm sorry for the late reply. It's not fixed. I've converted it as a discussion thread so other people who have this need could still discuss about it, and the original issue is automatedly closed. No worry about this and feel free to give deepin another try if you like :)

**Comment by meiyixiang** [comment]
I'm really sorry that I didn't see your problem until now. We will consider how to meet your demands in a better way. I have recorded the problem

**Comment by meiyixiang** [comment]
![image](https://user-images.githubusercontent.com/75475994/162919034-f3de6ad5-dd1f-4209-8d2c-16c8a1ba6a8b.png)

Hello, now in the latest version in extended screen mode you can choose to follow the mouse position or show it on the home screen and that will solve your problem
